src/main/java/net/minecraftforge/gradle/extrastuff/ReobfExceptor.java

```diff
@@ -24,6 +24,7 @@ public class ReobfExceptor {
     public File methodCSV;
     public File fieldCSV;
     public File excConfig;
+    public boolean copyEmptyDirectories; // Only for SpecialSource
```

Comment:
I don't like to add `copyEmptyDirectories` here. I think adding `copyEmptyDirectories` parameter to `ObfArtifact.generate` is better.